### PR TITLE
Feature/rename newdraft

### PR DESCRIPTION
**Pull request by rquazi**
### Description

 Rename "new draft" related items #535 to submission

### Related issues

all the references to components and routes containing string "newdraft" should be submission from this point

### Type of change

- [x ] Bug fix (non-breaking change which fixes an issue)

### Changes Made

- renamed all the file names, wizards, routes and tests containing name newdraft to submission.

### Testing

- [ ] Unit Tests
- [ ] Integration Tests
- [ ] Tests do not apply
- [ ] Needs testing (start an issue or do a follow up PR about it)

### CODE REVIEW

**Comment by blankdots** [comment]
it will be good to base this on `develop` branch as that is the target we are working on now
you can do this following: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request

**Comment by rquazi** [comment]
Sorry that's a mistake, I was working based on the develop branch. Not sure how it happened. I will fix this and will create a new PR.

**Comment by rquazi** [comment]
I have changed the base branch to develop for the PR and hopefully it is ready now for review.

**Comment by rquazi** [comment]
And about updating the changelog, should I add any release number or just add the changes under unreleased header?

**Comment by saulipurhonen** [comment]
> And about updating the changelog, should I add any release number or just add the changes under unreleased header?

This should go under "Unreleased" header since we just released new version.
Also, I'd add this change under "Changed" sub header since it's not a new feature.

**Comment by saulipurhonen** [comment]
There's a string "dispatch(createNewDraftFolder(folder))" in [architecture.md line 124](706/files#diff-4e03c430fc7f575c72fd989625acdc7d8d70fd20d3693c893d67d9dbd461ccdbR124)

Dispatch method should be `createNewDraftFolder`

Documentation for this is also outdated since `createSubmissionFolder` method expects 2-3 arguments instead of 1.

File: src/components/SubmissionWizard/WizardComponents/WizardStepper.tsx [hunk 1]
```diff
@@ -138,7 +138,7 @@ const WizardStepper = ({
   const params = useParams()
 
   const folderId = params.folderId
-  const newDraftPath = pathWithLocale(`newdraft/${folderId}`)
+  const newDraftPath = pathWithLocale(`submission/${folderId}`)
```
Comment (saulipurhonen) [inline src/components/SubmissionWizard/WizardComponents/WizardStepper.tsx:141]: I might be nitpicking but maybe the variable name could be something like `const submissionPath`?

**Comment by saulipurhonen** [comment]
Aside from comments above, looks good to me! :)

**Comment by rquazi** [comment]
I have added the changes mentioned by @saulipurhonen.

**Review by hannyle — APPROVED** [review]
I think we may need to rebase this PR onto `develop` branch again as there are some conflicts with `develop`.
Other than that, it looks good to me!

**Comment by rquazi** [comment]
ok, I will do that :)

**Review by blankdots — APPROVED** [review]
besides the rebase as Hang mentioned, everything else looks good

**Review by lilachic — APPROVED** [review]
LGTM, I agree on rebasing need :)

**Comment by rquazi** [comment]
now the conflicts have been resolved, should we merge this to develop :)

**Review by hannyle — APPROVED** [review]

**Pull request merged**